website/docs/compileas.md

```diff
@@ -12,9 +12,9 @@ compileas "value"
 * `C++` - Compile as a C++ source file.
 * `Objective-C` - Compile as an Objective-C source file.
 * `Objective-C++` - Compile as an Objective-C++ source file.
-* `Module` - Needs documentation
-* `ModulePartition` - Needs documentation
-* `HeaderUnit` - Needs documentation
+* `Module` - Compile as a C++20 module interface unit.
+* `ModulePartition` - Compile as a C++20 module interface partition.
+* `HeaderUnit` - Compile as a C++20 header unit.
```

Comment:
Shouldn't below "Availability"'s section should states that those parameter are only supported by msvc generator?

Reply:
Amended.